### PR TITLE
build: fix pd build by pinning tokio-util

### PR DESCRIPTION
**Pull request by conorsch**
Refs #1977.

### CODE REVIEW

File: pd/Cargo.toml [hunk 1]
```diff
@@ -55,7 +55,8 @@ directories = "4.0"
 # Temporarily pin due to https://github.com/penumbra-zone/penumbra/issues/1355
 tokio = { version = "=1.21.1", features = ["full"] }
 tokio-stream = "0.1"
-tokio-util = "0.7"
+# Temporarily pin due to https://github.com/penumbra-zone/penumbra/issues/1977
+tokio-util = "=0.7.4"
```
Comment (Darksonn) [inline pd/Cargo.toml:59]: If you bump Tokio to 1.22.0 or greater, then it should work.

Comment (conorsch) [reply pd/Cargo.toml:59]: Many thanks, will try that. There are a few out of date version pins that we may be able to remove, investigating...

**Comment by conorsch** [comment]
Removed several months-old version pins on the tokio deps. We're now using tokio v1.25 (from 1.21.1).

**Comment by conorsch** [comment]
Closes #1977.

**Pull request merged**